dom/app.js

```diff
@@ -0,0 +1,30 @@
+const btnCreate = document.querySelector('.btn-main');
+const btnToggle = document.querySelector('.btn-toggle');
+const btnRemove = document.querySelector('.btn-remove');
+
+btnCreate.addEventListener('click', () => {
+    const input = document.querySelector('.input-main');
+    const list = document.querySelector('ul');
+
+    list.insertAdjacentHTML(
+        'afterbegin',
+        `<li>${input.value}</li>`
+    );
+    input.value = '';
+});
+
+btnRemove.addEventListener('click', () => {
+    const lastItem = document.querySelector('li:last-child');
+    lastItem.remove();
+});
+
+btnToggle.addEventListener('click', () => {
+    const listContainer = document.querySelector('.list-container');
+    if (listContainer.style.display === 'none') {
+        btnToggle.textContent = 'Hide List';
+        listContainer.removeAttribute('style');
+    } else {
```

Comment:
You know me. 
I hate else blocks.
Return first.

Reply:
Done